test/controllers/NCTSMessageControllerSpec.scala

```diff
@@ -200,6 +203,7 @@ class NCTSMessageControllerSpec extends SpecBase with ScalaCheckPropertyChecks w
 
           val result = route(application, request).value
 
+          contentAsString(result) mustEqual "DocNumHEA5 was empty"
```

Comment:
The error message generated could be changed to the following:<br>`The element 'DocNumHEA5' must contain a value.`